documentation/airport-guides/eggw.md

```diff
@@ -0,0 +1,109 @@
+
+45,561,700 passengers handled in 2016-17
+285,969 aircraft movements in 2016-17
+
+08L/26R 2,565m, Asphalt
+08R/26L 3,375m, Asphalt (primary operational runway) 
+
+##### London City Airport, EGLC
+London City Airport is an international airport located in the Royal Docks near Canary Wharf. It has an extremely short runway; Short Takeoff and Landing operations are in use. The airport was officially opened by Queen Elizabeth II in November 1987. The airport is mainly used by business travellers due to its proximity to the finanical district; various business airlines such as British Airways, Lufthansa and KLM operate flights to London City. British Airways operates a flagship route from London City, BA1, from London City to Shannon to New York with an A318.
+
+4,538,813 passengers handled in 2015-16
+85,169 aircraft movements in 2015-16
+
+09/27 1,508m, Asphalt
+
+##### London Stansted Airport, EGSS
+London Stansted Airport is an international airport located at Stansted Mountfitchet, Essex. It is 48km northeast of Central London. The airport servers over 170 destinations across Europe and is a base for many low-cost airlines, most notably Ryanair. The airport is owned by the Manchester Airports Group, which operates Manchester Airport and East Midlands Airport. The airport is a focus city for easyJet, Jet2.com, Ryaniar, Thomas Cock and TUI.
+
+24,320,071 passengers handled in 2015-16
+180,430 aircraft movements in 2015-16
+
+04/22 3,049m, Grooved asphalt
+
+##### London Southend Airport, EGMC
+London Southend Airport is an international airport 68km from Central London. The airport serves very few airlines, however it has been increasing passenger numbers over the years. British low-cost carriers Flybe and easyJet have been one of the reasons behind this increase in passenger numbers as they have been starting new routes from Southend recently.
+
+05/23 1,856m, Asphalt
+
+##### London Farnborough Airport, EGLF
+Farnborough Airport is a business and general aviation airport in Hampshire. The airfield is the home of the Farnborough Airshow which is held every two years.
+
+06/2 2,440m, Asphalt
+
+##### London Biggin Hill Airport, EGKB
+Biggin Hill Airport is a general aviation airport 22km south-east of Central London. It was formerly an RAF station. Radar air traffic control services are given by Thames Radar at the LTCC and VFR ATC are provided by the airport itself. There are no commercial air services to or from Biggin Hill; all requests for such commercial services have been rejected by the council and no scheduled nor fare-paying passenger services are permitted to operate here.
+
+03/21 1,820m, Tarmac
+
+
+### Videomap
+Provided with the videomap is a generic videomap for the London Terminal Manoeuvring Area. The extended centrelines and runways of all major airports (Heathrow, Gatwick, Stansted, Luton and City) are displayed, along with the positions of important VORs. CTAs and ATZs of all the major airports are displayed too.

```

Comment:
Those aren't the only airports shown on the video map

Reply:
I can make it less ambiguous, but the main focus there was _major_ airports (Farnborough and BH aren't really major)